rover_gui/src/QArbitration/QArbitration.hpp

```diff
@@ -54,14 +54,18 @@ class QArbitration : public QWidget
   public:
     QArbitration(std::shared_ptr<rclcpp::Node> guiNode_, QWidget* parent_);
 
+  signals:
+    void joyDemuxStatusCallbackSignal(const rover_msgs::msg::JoyDemuxStatus::SharedPtr msg_);
+    void driveTrainDemuxStatusCallbackSignal(const rover_msgs::msg::DrivetrainArbitration::SharedPtr msg_);
+
   private slots:
     void onControllerComboChanged(eControllerType controller_, int index_);
     void onDriveTrainComboChanged(int index_);
+    void joyDemuxStatusCallback(const rover_msgs::msg::JoyDemuxStatus::SharedPtr msg_);
```

Comment:
Keep slot naming consistent to avoid mixing up with ros CB:
```suggestion
    void onjJyDemuxStatusChanged(const rover_msgs::msg::JoyDemuxStatus::SharedPtr msg_);
```